scrape/manager_test.go

```diff
@@ -336,6 +336,7 @@ scrape_configs:
 
 func TestManagerTargetsUpdates(t *testing.T) {
 	m := NewManager(nil, nil)
+	defer m.Stop()
```

Comment:
From a readability perspective it might be clearer to put this line right after `go m.Run(ts)`, since it's related to that and not needed for the scrape managers in the other tests that are constructed, but where `Run()` is never called.

Reply:
yes. it looks like better to put stop after the run method

Reply:
> From a readability perspective it might be clearer to put this line right after `go m.Run(ts)`, since it's related to that and not needed for the scrape managers in the other tests that are constructed, but where `Run()` is never called.

updated.